cts/CTSlab.py

```diff
@@ -30,33 +30,25 @@
 sys.path.insert(0, pdir) # So that things work from the source directory
 
 try:
-    from cts.CTSvars    import *
-    from cts.CM_ais     import *
-    from cts.CM_lha     import crm_lha
-    from cts.CTSaudits  import AuditList
-    from cts.CTStests   import TestList
+    from cts.CTSvars      import *
+    from cts.CM_ais       import *
+    from cts.CM_lha       import crm_lha
+    from cts.CTSaudits    import AuditList
+    from cts.CTStests     import TestList
     from cts.CTSscenarios import *
     from cts.logging      import LogFactory
-
-except ImportError:
-    sys.stderr.write("abort: couldn't find cts libraries in [%s]\n" %
+except ImportError as e:
+    sys.stderr.write("abort: %s\n" % e)
```

Comment:
Correct, for example adding "from cts.foo import *":

abort: No module named foo
check your install and PYTHONPATH; couldn't find cts libraries in:
/home/kgaillot/src /home/kgaillot/src/pacemaker /home/kgaillot/src/pacemaker/cts /usr/lib64/python27.zip /usr/lib64/python2.7 /usr/lib64/python2.7/plat-linux2 /usr/lib64/python2.7/lib-tk /usr/lib64/python2.7/lib-old /usr/lib64/python2.7/lib-dynload /usr/lib64/python2.7/site-packages /usr/lib64/python2.7/site-packages/gtk-2.0 /usr/lib/python2.7/site-packages